### DISCUSSION TITLE
Transit engine export key api missing?

### DISCUSSION THREAD

**Discussion by youqingz** [rajanadar/VaultSharp#198] [Q&A] [answered]
I couldn't find the method on ITransitSecretsEngine for the export key API.
https://www.vaultproject.io/api/secret/transit#export-key
Is this not supported?

**Comment by rajanadar** [comment, marked as answer]
@youqingz  It doesn't exist right now. I'll add it and release a new package over the weekend.

**Reply by rajanadar** [reply]
@youqingz  This is available in https://www.nuget.org/packages/VaultSharp/1.6.5.1 now
